app/views/admin/operational_fields/index.html.erb

```diff
@@ -1,20 +1,30 @@
-<% page_title "Fields of operation" %>
-<section>
-  <h1>Fields of operation</h1>
-  <%= link_to "Add field of operation", new_admin_operational_field_path, class: "btn btn-default new_resource add-top-margin", title: "Add field of operation" %>
+<% content_for :page_title, "Fields of operation" %>
+<% content_for :title, "Fields of operation" %>
 
-  <table class="fields_of_operation table table-striped table-bordered add-top-margin">
-    <thead>
-      <tr class="table-header">
-        <th width="30%">Name</th>
-      </tr>
-    </thead>
-    <tbody>
-      <% @operational_fields.each do |operational_field| %>
-        <%= content_tag_for(:tr, operational_field) do %>
-          <td class="name"><%= link_to operational_field.name, edit_admin_operational_field_path(operational_field), title: "Edit field of operation #{operational_field.name}" %></td>
-        <% end %>
-      <% end %>
-    </tbody>
-  </table>
-</section>
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <%= render "govuk_publishing_components/components/button", {
+      text: "Add field of operation",
+      href: new_admin_operational_field_path,
+      secondary_solid: true,
+      margin_bottom: 8,
+    } %>
+
+    <%= render "govuk_publishing_components/components/table", {
+      head: [
+        {
+          text: "Name"
+        },
+      ],
+      rows:
+        @operational_fields.map do |operational_field|
+          [
+            {
+              text: tag.span("Edit field of operation", class: "govuk-visually-hidden") +
+              link_to(operational_field.name, edit_admin_operational_field_path(operational_field), class:"govuk-link")
```

Comment:
```suggestion
              text:  link_to("#{tag.span("Edit field of operation", class: "govuk-visually-hidden")} 
 #{operational_field.name}", edit_admin_operational_field_path(operational_field), class:"govuk-link")
```

Link should include the visually hidden text

Reply:
ah thanks for the review. Good catch. 

Annoyingly i've had to pass it in as html and sanitise it. If i call sanitise on this ☝️ it's already escaped the span.